platform/broadcom/sonic-platform-modules-dell/debian/platform-modules-s6100.install

```diff
@@ -4,6 +4,7 @@ common/dell_i2c_utils.sh usr/local/bin
 common/io_rd_wr.py usr/local/bin
 common/fstrim.timer etc/systemd/system
 common/fstrim.service etc/systemd/system
+common/platform_reboot usr/share/sonic/device/x86_64-dell_s6100_c2538-r0
```

Comment:
files in device folder are generated at build time. please check the file directly to the folder. 

currently approach only install the file at the run time.

Reply:
Addressed in new commit.